src/components/calcite-filter/calcite-filter.tsx

```diff
@@ -0,0 +1,93 @@
+import { Component, Element, Event, EventEmitter, Host, Prop, h } from "@stencil/core";
+import { debounce, forIn } from "lodash-es";
+
+const filterDebounceAmount = 250; // milliseconds
+
+@Component({
+  tag: "calcite-filter",
+  styleUrl: "calcite-filter.scss",
+  shadow: true
+})
+export class CalciteFilter {
+  // --------------------------------------------------------------------------
+  //
+  //  Properties
+  //
+  // --------------------------------------------------------------------------
+
+  @Prop() textLabel: string;
+
+  @Prop() textPlaceholder: string;
+
+  @Prop() data: object[];
```

Comment:
I forgot, but do we want to handle exact vs like matches?

Reply:
partial matching. The exact matching is an app feature that I provided a demo to Adelheide for.

Reply:
ok